LocalPackages/NewTabPage/Sources/NewTabPage/NewTabPageActionsManager.swift

```diff
@@ -56,6 +57,17 @@ public extension NewTabPageScriptClient {
             userScript.broker?.push(method: method, params: params, for: userScript, into: webView)
         }
     }
+
+    /**
+     * Convenience method to push a message with specific parameters to the user script
+     * associated with the given `webView`.
+     */
+    func pushMessage(named method: String, params: Encodable?, to webView: WKWebView) {
```

Comment:
By default all messages are sent to all New Tab Pages (in all open application windows). But we now have a case when we want to send a message only to a specific NTP, which is the message to open settings panel that needs to be handled only in the currently active webView. Hence this addition.